docs/source/manual/testing.rst

```diff
@@ -365,24 +365,27 @@ By creating a DropwizardTestSupport instance in your test you can manually start
     public class LoginAcceptanceTest {
 
         public static final DropwizardTestSupport<TestConfiguration> SUPPORT =
-                new DropwizardTestSupport<TestConfiguration>(MyApp.class, ResourceHelpers.resourceFilePath("my-app-config.yaml"));
+                new DropwizardTestSupport<TestConfiguration>(MyApp.class,
+                    ResourceHelpers.resourceFilePath("my-app-config.yaml"),
+                    ConfigOverride.config("server.applicationConnectors[0].port", "0") // create random port
```

Comment:
OK, got it.
I would like to use the original config file of the application to avoid maintaining two config files.
That's why I found overriding more charming.

Maybe just change the comment to make clear, that this is optional?